### ISSUE TITLE
 Dev.to Posts View counts in Dashboard are not changing 

### ISSUE THREAD

**Issue by veselinastaneva** [forem/forem#9398]
**Describe the bug**

Posts View counts in Dashboard are not changing in stats for a couple of days. I have entered my post from different devices and the View count didn't change at all. The number is updated sometimes but it usually gets stuck for 3-4 days

**To Reproduce**

1. Go to dev.to > your profile > Dashboard > Views
2. Enter your last post from another dev. to profile
3. Hard Refresh your dev.to dashboard
4. the Views count doesn't change for days

**Expected behavior**

- OS: Windows
- Browser: Chrome
- Version: Version 83.0.4103.116 (Official Build) (64-bit)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by AlbertoPdRF** [comment]
I believe this is a duplicate of #8425.

**Comment by rhymes** [comment]
Thanks @AlbertoPdRF!
